### PR TITLE
Fixed : Can add application to folder multiple times 

### LINKED ISSUE #1319: Can add application to folder multiple times
### Version Information

v0.8.1

### Environment

Docker

### What is the expected behaviour?

Add some application to an folder. But disallow if it is in a folder.
Also: A application can only be in one folder. Auto remove it from the current folder.

### What is the current behaviour?

I can add application to multiple folder also i can add the same application in the same folder multiple times.

### How to reproduce the issue?

1. create a application
2. add to multiple folder or same folder

### Screenshots or Screencast

grafik

### PR DESCRIPTION
**Pull request by saisumanthkumar**
Fixes #1319 

heroku-deploy : https://tooljet-s.herokuapp.com/

### CODE REVIEW

**Comment by saisumanthkumar** [comment]
Yeah Now good to go, please review

**Comment by saisumanthkumar** [comment]
@Navaneeth-pk @arpitnath please check this

File: frontend/src/HomePage/Folders.jsx [hunk 1]
```diff
@@ -29,6 +29,18 @@ export const Folders = function Folders({
       });
       return;
     }
+    
+    for(let i = 0; i < folders.length; i++){
```
Comment (Navaneeth-pk) [inline frontend/src/HomePage/Folders.jsx:33]: I think we should handle this at the entity level or in DB schema. 
@akshaysasidrn what do you think?

Comment (saisumanthkumar) [reply frontend/src/HomePage/Folders.jsx:33]: @Navaneeth-pk sorry I can't get you, what do I need to change ?

Comment (akshaysasidrn) [reply frontend/src/HomePage/Folders.jsx:33]: Yeah, I agree. 
Basically at the backend at server we need to check if the app is already added to the folder and raise an error if so.

Comment (saisumanthkumar) [reply frontend/src/HomePage/Folders.jsx:33]: ok,shall I work on backend ? @Navaneeth-pk @akshaysasidrn

Comment (Navaneeth-pk) [reply frontend/src/HomePage/Folders.jsx:33]: @saisumanthkumar sure 👍

Comment (saisumanthkumar) [reply frontend/src/HomePage/Folders.jsx:33]: @Navaneeth-pk so can I close this PR now

Comment (Navaneeth-pk) [reply frontend/src/HomePage/Folders.jsx:33]: @saisumanthkumar no, you can use the same PR. Just revert the old changes and make changes in the backend instead.

Comment (saisumanthkumar) [reply frontend/src/HomePage/Folders.jsx:33]: @Navaneeth-pk ok

**Pull request closed without merging**